.travis.yml

```diff
@@ -17,12 +17,10 @@ jdk:
   - oraclejdk8
 
 install:
-  - mvn clean install -DskipTests -Dassembly.skipAssembly=true > /dev/null
+  - bash .travis_install.sh
 
 script:
-  - cd $PINOT_MODULE
-  - cp ../.travis_cmd.sh .
-  - bash .travis_cmd.sh
+  - bash .travis_test.sh
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
```

Comment:
But what if they change the report uploader code and the old version is not compatible with the codecov website? Also, this part of the code will only run on Travis machine (basically a sand box), and our code is fully open-sourced, so I don't think it can cause any security issue in our side.